docs_src/microservices/device/services/device-gpio/GettingStarted.md

```diff
@@ -0,0 +1,174 @@
+                    "path": "/api/v3/device/name/GPIO-Device01/Power",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "Power",
+                            "valueType": "Bool"
+                        }
+                    ]
+                },
+                {
+                    "name": "LED",
+                    "set": true,
+                    "path": "/api/v3/device/name/GPIO-Device01/LED",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "LED",
+                            "valueType": "Bool"
+                        }
+                    ]
+                },
+                {
+                    "name": "Switch",
+                    "get": true,
+                    "path": "/api/v3/device/name/GPIO-Device01/Switch",
+                    "url": "http://edgex-core-command:59882",
+                    "parameters": [
+                        {
+                            "resourceName": "Switch",
+                            "valueType": "Bool"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+    ```
+Use the `curl` response to get the command URLs (with device and command ids) to issue commands to the GPIO device via the command service as shown above. You can also use a tool like `Postman` instead of `curl` to issue the same commands.
+
+### Direction setting with sysfs
+When using sysfs, the operations to access and "read" or "write" the GPIO pins are to:
+
+1. Export the pin
+2. Set the direction (either IN or OUT)
+3. Read the pin input or write the pin value based on the direction
+4. Unexport the pin 
+
+When using sysfs, setting the direction causes the value to be reset.  Therefore, this implementation only sets the direction on opening the line to the GPIO.  After that, it is assumed the same direction is used while the pin is in use and exported.
+
+The direction is set by an optional attribute in the device profile called `defaultDirection`.  It can be set to either "in" or "out".  If it is not set, the default direction is assumed to be "out".
+
+!!! example - "Example - GPIO resource in device profile"
+    ``` yaml
+      -
+        name: "LED"
+        isHidden: false
+        description: "mocking LED"
+        attributes: { line: 27, defaultDirection: "out" }
+        properties:
+          valueType: "Bool"
+          readWrite: "W"
+    ```
+
+!!! note
+    The direction should not be confused with the device profile's read/write property.  If you set the defaultDirection to "in" but then set the readWrite property to "RW" or "W", any attempt to write to the pin will result in a "permission denied" error.  For consistency sake, when your defaultDirection is "in" set readWrite to "R" only.
```

Comment:
Do we want these to be in quotes or in-line code?
```suggestion
    The direction should not be confused with the device profile's read/write property.  If you set the defaultDirection to `in` but then set the readWrite property to `RW` or `W`, any attempt to write to the pin will result in a `permission denied` error.  For consistency sake, when your defaultDirection is `in` set readWrite to `R` only.
```

Reply:
I like in-line.